test/unit/lib/much-rails/call-method_tests.rb

```diff
@@ -0,0 +1,61 @@
+require "assert"
+
+require "much-rails/call-method"
+
+module MuchRails::CallMethodTest
+  class UnitTests < Assert::Context
+    desc "MuchRails::CallMethod"
+    subject { unit_class }
+
+    let(:unit_class) { MuchRails::CallMethod }
+
+    should "include MuchPlugin" do
+      assert_that(subject).includes(MuchPlugin)
+    end
+  end
+
+  class ReceiverTests < UnitTests
+    desc "receiver"
+    subject { receiver_class }
+
+    setup do
+      MuchStub.tap(subject, :new) { |new_record|
+        @new_class_method_called = true
+
+        MuchStub.(new_record, :call) {
+          @call_instance_method_called = true
+        }
+      }
+    end
+
+    let(:receiver_class) {
+      Class.new do
+        include MuchRails::CallMethod
+      end
+    }
+
+    should have_imeths :call
+
+    should "call #new and #call" do
+      subject.call
+
+      assert_that(@new_class_method_called).equals(true)
+      assert_that(@call_instance_method_called).equals(true)
+    end
+  end
+
+  class ReceiverWithCallNotImplemented < UnitTests
+    desc "with call not implemented"
+    subject { receiver_class }
+
+    let(:receiver_class) {
+      Class.new do
+        include MuchRails::CallMethod
+      end
+    }
+
+    should "not implement its call method" do
+      assert_that(-> { subject.call }).raises(NotImplementedError)
+    end
```

Comment:
```suggestion
    should "not implement its #call method" do
      assert_that(-> { subject.call }).raises(NotImplementedError)
    end
```
Since we are specifically talking about a method.